packages/ngtools/webpack/package.json

```diff
@@ -28,14 +28,14 @@
     "webpack-sources": "1.3.0"
   },
   "peerDependencies": {
-    "@angular/compiler-cli": ">=5.0.0 <8.0.0 || ^7.0.0-beta.0",
-    "typescript": ">=2.4.0 < 3.3",
+    "@angular/compiler-cli": ">=5.0.0 <9.0.0 || ^7.0.0-beta.0 || ^8.0.0-beta.0",
+    "typescript": ">=2.4.0 < 3.4",
```

Comment:
Should we continue supporting so many versions of typescript without proper testing?

As a side note, framework dropped support for TypeScript versions prior to 3.3 in Version 8.

Reply:
That's a good question. 

@hansl I think you checked what the supporter Angular versions were for CLI v8, can you advise?

Reply:
@filipesilva I was referring to supported TypeScript versions, but yeah same thing applies for supported Angular versions.

Reply:
They are linked together really. We only support older TS because of Angular.

Reply:
Definitely one version down.  Upwards was the open question.

Reply:
I think we should update this as per @alexeagle PR https://github.com/angular/angular-cli/pull/13910.
```
"@angular/compiler-cli": ">=6.0.0 <9.0.0 || ^8.0.0-beta.0",
 "typescript": ">=2.7 < 4",
```

Reply:
TypeScript does not follow semver so using `<4` opens us up to breaking changes. Our policy is to move lockstep with their minor. Is there a reason to use `<4` this time that I am unaware of?

Reply:
@filipesilva, apologies it should be;

```
 "typescript": ">=2.7 < 3.4",
```